Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN npm install
 FROM node:8.11.3-alpine
 WORKDIR /opt/finance-portal-backend-service
 
-COPY --from=builder /opt/finance-portal-backend-service ./src
-COPY ./src ./src
+COPY --from=builder /opt/finance-portal-backend-service /opt/finance-portal-backend-service
+COPY ./src /opt/finance-portal-backend-service
 
-CMD ["node", "src/index.js"]
\ No newline at end of file
+CMD ["node", "/opt/finance-portal-backend-service/index.js"]
```

Comment:
I'm fairly sure your assessment of the cause of this error is correct, but I'm unsure how this fixes it. Could you clarify?

Reply:
I logged the handlers loaded here: https://github.com/mojaloop/finance-portal-backend-service/blob/master/src/server.js#L138

and found out that it wasn't loading any files. It works locally, but not on the docker image, because `"./handlers/*.js"` doesn't exists, due to the current directory being "/src". The correct route should be "./src/handlers/*js", but that won't work if you try to run the portal locally. So, just playing around with the directories did the trick.

Reply:
@Kalaston I think your reply refers to something different than what @msk- mentions, as he points to Dockerfile.

@msk- The path of the source code has been moved to `/opt/finance-portal-backend-service/` in order to align with the rest of the Mojaloop repositories and also with its ";licensce scanner" tool that expects the source to exist in in that directory explicitly and if it doesn't find anything will fail. So that is why the command had to change as well.